apps/mark/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -87,13 +87,13 @@ function UpdatePollsButton({
   const explanationText = (() => {
     switch (pollsTransition) {
       case 'open_polls':
-        return `After polls are opened, the machine will be ready for voters to mark and print ballots.`;
+        return `After polls are opened, voters will be able to mark ballots.`;
       case 'pause_voting':
-        return `After voting is paused, the machine will no longer allow voters to mark and print ballots until voting is resumed.`;
+        return `After voting is paused, voters will not be able to mark ballots until voting is resumed.`;
       case 'resume_voting':
-        return `After voting is resumed, the machine will be ready for voters to mark and print ballots.`;
+        return `After voting is resumed, voters will be able to mark ballots.`;
       case 'close_polls':
-        return `After polls are closed, the machine will no longer allow voters to mark and print ballots. Polls cannot be opened again after being closed.`;
+        return `After polls are closed, voters will no longer be able to mark ballots. Polls cannot be opened again after being closed.`;
```

Comment:
I like this permutation of the copy! Not a big deal at all but the screenshots in the PR are out-of-date now, and I was going to suggest exactly this - standardizing on "After [transition], voters [will / will not] be able to [action]." Clearer and more succinct than the "the machine will" alternatives

Reply:
Screenshots in the overview are correct - I just deleted the comment with the older snapshots.